papermill/tests/test_inspect.py

```diff
@@ -52,7 +52,12 @@ def click_context():
     ],
 )
 def test_inspect_notebook(name, expected):
-    assert inspect_notebook(str(name)) == expected
+    assert inspect_notebook(name) == expected
+
+
+def test_str_path():
+    expected = {"msg": {"name": "msg", "inferred_type_name": "None", "default": "None", "help": ""}}
```

Comment:
Nit: Can we fix the formatting on this value?

Reply:
Sure. This was copied verbatim from [higher in the file](https://github.com/nteract/papermill/blob/main/papermill/tests/test_inspect.py#L30). What would you like it changed to and would you like me to fix the formatting throughout the file to match?

Reply:
Surprisingly, this is exactly how black formats this dict. I think we have to leave it as is. 😫

Reply:
I've been living in JavaScript land for too long, haven't I? All good then!